install.sh

```diff
@@ -39,6 +39,43 @@ get_os_version() {
     OS_VERSION=$(egrep '^(VERSION_ID)=' /etc/os-release | sed 's/"//g' | cut -f2 -d"=")
 }
 
+# Function to check system prerequisites
+pre-check-installation(){
+    cpu_threshold='90'
+    mem_threshold='1024'
+    disk_threshold='200'
```

Comment:
The disk threshold should be much higher, is it MB?  the images, etc will account for more MBs

Reply:
yes, 200Mb free after OS installation. I mean, you have your VM with 500mb, ubuntu takes 250, you have 250 free which is > than this threadhold -> you can install it

If not which should be the minimum threshold are you thinking?

Reply:
after talking about it, let's change to 2048 mb (2gb)